src/main/java/org/alliancegenome/curation_api/jobs/executors/ParalogyExecutor.java

```diff
@@ -38,20 +45,18 @@
 
 			List<Long> paralogyIdsLoaded = new ArrayList<>();
 			BackendBulkDataProvider dataProvider = BackendBulkDataProvider.valueOf(fms.getFmsDataSubType());
-			// String dataProviderAbbreviation = fms.getFmsDataSubType();
-			// List<Object[]> orthoPairsBefore =
-			// generatedOrthologyService.getAllOrthologyPairsBySubjectGeneDataProvider(dataProviderAbbreviation);
-			// log.debug("runLoad: Before: total " + orthoPairsBefore.size());
+			List<Long> paralogyPairsBefore = geneToGeneParalogyService.getAllParalogyPairIdsBySubjectGeneDataProvider(dataProvider);
+			Log.debug("runLoad: Before: total " + paralogyPairsBefore.size());
 
 			bulkLoadFileDAO.merge(bulkLoadFile);
 
 			BulkLoadFileHistory history = new BulkLoadFileHistory(paralogyData.getData().size());
-
 			createHistory(history, bulkLoadFile);
-			runLoad(geneToGeneParalogyService, history, dataProvider, paralogyData.getData(), paralogyIdsLoaded);
-
-			// runCleanup(history, fms.getFmsDataSubType());
+			boolean success = runLoad(geneToGeneParalogyService, history, dataProvider, paralogyData.getData(), paralogyIdsLoaded);
 
+			if (success) {
+				runCleanup(geneToGeneParalogyService, history, fms.getFmsDataSubType(), paralogyPairsBefore, paralogyIdsLoaded,fms.getFmsDataType(), bulkLoadFile.getMd5Sum(),false);
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>',' is not followed by whitespace.